doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -340,6 +341,12 @@ Configure the module to enable a run-time short key-derivation key check when
 deriving a key by X963KDF.
 See NIST SP 800-131Ar2 for details.
 
+=item B<-x942kdf_key_check>
```

Comment:
We probably need a history section near the end since these options got added in different OpenSSL versions

Reply:
=/ true, unless we choose to take this into 3.4 as I see it this is a miss when similar key-check was added to all other KDFs.

Also see another indicator fixup submitted in https://github.com/openssl/openssl/pull/25517

Reply:
Could be a separate PR..

Reply:
Please see https://github.com/openssl/openssl/pull/25546/files